# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleLoadRequest.cpp
# Commit: 1b987903eab3
# Full Hash: 1b987903eab3fd39c65e6cfd87e3027eb10e616b
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-09-24 18:03 +0000
# Regressor Bug: 1968890
# File Overlap Count: 5
# Overlapping Files: js/loader/ModuleLoadRequest.cpp, js/loader/ModuleLoaderBase.cpp, js/loader/ModuleLoadRequest.h, js/loader/ModuleLoaderBase.h, dom/script/ScriptLoader.cpp
# Description:
#   Bug 1968890 - Part 7: Add Cancel method. r=jonco
# ==============================================================================

diff -r 75ae330e1b90 -r 1b987903eab3 js/loader/ModuleLoadRequest.cpp
--- a/js/loader/ModuleLoadRequest.cpp	Wed Sep 24 18:03:30 2025 +0000
+++ b/js/loader/ModuleLoadRequest.cpp	Wed Sep 24 18:03:30 2025 +0000
@@ -74,22 +74,6 @@
   return mModuleScript->HasParseError() || mModuleScript->HasErrorToRethrow();
 }
 
-void ModuleLoadRequest::Cancel() {
-  if (IsCanceled()) {
-    return;
-  }
-
-  if (IsFinished()) {
-    return;
-  }
-
-  ScriptLoadRequest::Cancel();
-
-  mModuleScript = nullptr;
-  mReferrerScript = nullptr;
-  mModuleRequestObj = nullptr;
-}
-
 void ModuleLoadRequest::SetReady() {
   MOZ_ASSERT(!IsFinished());
 